pkg/processor/runtime/rpc/connection/abstract.go

```diff
@@ -88,8 +87,39 @@ func (bc *AbstractConnectionManager) UpdateStatistics(durationSec float64) {
 	bc.Configuration.Statistics.DurationMilliSecondsSum += uint64(durationSec * 1000)
 }
 
+func (bc *AbstractConnectionManager) GetConfig() ManagerConfigration {
```

Comment:
off-topic question - why is the shorthand `bc` ? 
It can be `m` / `cm` / `acm`, but no need to change in this PR.

Reply:
Not sure tbh